packages/iris-grid/src/IrisGridProxyModel.ts

```diff
@@ -317,7 +317,7 @@ class IrisGridProxyModel extends IrisGridModel implements PartitionedGridModel {
     ) {
       if (partitionConfig.mode === 'keys') {
         modelPromise = this.originalModel
-          .partitionBaseTable()
+          .partitionKeysTable()
```

Comment:
This should still be returning the `partitionBaseTable()`. It is kind of confusing that the mode is `'keys'` and we're getting the base table here, but we want to show the user the base table when they're in `keys` mode; we use the `keysTable` for getting the options for the partition selector itself.

```suggestion
          .partitionBaseTable()
```